### PR TITLE
Add some additional methods to the taskcluster module

### PR DESCRIPTION
**Pull request by marco-c**
To support #304.

E.g. to get all tasks belonging to a given push:
```
decision_task_id = taskcluster.find_task_id(f"gecko.v2.{branch}.revision.{rev}.taskgraph.decision")
decision_task_data = taskcluster.get_task(decision_task_id)  # could avoid this request and always assume the decision_task_id is the same as the group ID.
tasks = get_tasks_in_group(decision_task_data["taskGroupId"])
```

### CODE REVIEW

File: mozci/util/taskcluster.py [hunk 1]
```diff
@@ -73,3 +75,26 @@ def get_index_url(index_path):
...
+def get_tasks_in_group(group_id):
+    tasks = []
+
+    def _save_tasks(response):
+        tasks.extend(response["tasks"])
+
+    queue = taskcluster_config.get_service("queue")
+    queue.listTaskGroup(group_id, paginationHandler=_save_tasks)
+
+    return tasks
```
Comment (marco-c) [inline mozci/util/taskcluster.py:100]: I could have implemented this using requests, but this is way easier.

File: mozci/util/taskcluster.py [hunk 2]
```diff
@@ -6,11 +6,13 @@
 import functools
 
 import taskcluster_urls as liburls
+from taskcluster.helper import TaskclusterConfig
```
Comment (ahal) [inline mozci/util/taskcluster.py:9]: You'll need to `poetry add` the taskcluster module.

File: mozci/util/taskcluster.py (hunk 1 above)
Comment (ahal) [reply mozci/util/taskcluster.py:100]: It's not that hard :p 
https://searchfox.org/mozilla-central/source/taskcluster/taskgraph/util/taskcluster.py#327

But agree that using the API seems nicer. Should we convert everything to use the API?

**Review by ahal — CHANGES_REQUESTED** [review]

File: mozci/util/taskcluster.py (hunk 1 above)
Comment (marco-c) [reply mozci/util/taskcluster.py:100]: > It's not that hard :p

Yeah, but it is cleaner (i.e. less code) this way :P

> But agree that using the API seems nicer. Should we convert everything to use the API?

Yeah, I'll file it as a good first issue.

File: mozci/util/taskcluster.py (hunk 2 above)
Comment (marco-c) [reply mozci/util/taskcluster.py:9]: Oops, I guess I was lucky as `adr` depends on `taskcluster`

File: mozci/util/taskcluster.py (hunk 1 above)
Comment (marco-c) [reply mozci/util/taskcluster.py:100]: I filed https://github.com/mozilla/mozci/issues/306.

**Review by ahal — APPROVED** [review]

**Pull request merged**